main.js

```diff
@@ -11,13 +11,18 @@ const BOARD_SIZE = 14;
 const HALF_BOARD_SIZE = BOARD_SIZE / 2;
 const SPHERE_RADIUS = 0.4;
 const SPHERE_WIDTH_SEGMENT = 32;
-const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 2;
+const SPHERE_HEIGHT_SEGMENT = SPHERE_WIDTH_SEGMENT / 3;
+const WINNING_NUMBER_OF_STONES = 5;
 
 const scene = new THREE.Scene();
 let currentPlayer = 'black';
+let currentPlayerLabel = null;
+let winnerAnnouncement = null;
+let restartButtonLabel = null;
+let stonesInScene = [];
 
 function initializeGameBoard() {
-    const gameBoard = Array(BOARD_SIZE).fill().map(() => Array(BOARD_SIZE).fill(null));
+    const gameBoard = Array(BOARD_SIZE + 1).fill().map(() => Array(BOARD_SIZE + 1).fill(null));
```

Comment:
I'm seeing the same changes in both PRs, which isn't ideal for review. I forgot about some of the dyanmics involved when using multiple branches in this way, so it's my fault. Let's chat about it at the next meeting